packages/terra-framework-docs/src/terra-dev-site/doc/tabs/example/ClosableTab.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import classNames from 'classnames/bind';
+import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import Tabs from 'terra-tabs';
+import TabContentTemplate from './TabContentTemplate';
+import styles from './common/TabExample.module.scss';
+
+const cx = classNames.bind(styles);
+
+const ClosableTab = () => {
+  const tabsExample = [];
+  const [tabs, setTabs] = useState(tabsExample);
+  const [activeKey] = useState();
```

Comment:
```suggestion
  const [activeKey] = useState();
```
I think we don't need this state here !!

Reply:
removed the active key from the closable tab example.